sdk/core/azure-core-http-netty/src/main/java/com/azure/core/http/netty/NettyAsyncHttpClientBuilder.java

```diff
@@ -180,6 +182,22 @@ public NettyAsyncHttpClientBuilder configuration(Configuration configuration) {
         return this;
     }
 
+    /**
+     * Disables deep copy of response {@link ByteBuffer} into a heap location that is managed by this client as
+     * opposed to the underlying netty library which may use direct buffer pool.
+     * <br>
+     * <b>
+     * Caution: Disabling this is not recommended as it can lead to data corruption if the downstream consumers
+     * of the response do not handle the byte buffers before netty releases them.
+     * </b>
```

Comment:
It would be good to include code here to demonstrate how to use the buffer correctly if the user chooses to disable this, and to also talk about how this will benefit throughput and memory pressure.

Reply:
I will add more docs for this.